tests/process_monitor/test_comm_from_server.py

```diff
@@ -690,9 +789,9 @@ def test_MantarrayProcessesMonitor__check_and_handle_server_to_main_queue__handl
     invoke_process_run_and_check_errors(monitor_thread)
 
     actual_log_message = mocked_monitor_logger_info.call_args[0][0]
-    assert expected_okc_item in actual_log_message
-    assert expected_fw_item in actual_log_message
-    assert expected_da_item in actual_log_message
+    assert str(expected_okc_item) in actual_log_message
```

Comment:
Might be missing it, but it doesn't look like the redaction is ever made, so now this is just asserting that the unredacted message is in the log

Reply:
I'm confused about what function to call to complete the redaction